.packaging/debian/control

```diff
@@ -0,0 +1,383 @@
+Source: gnuradio
```

Comment:
I know there was some discussion on chat about keeping these files separate from the main repo, but we'll never be able to keep this file in sync with the master (or whatever) branch if we do that. So I'm glad this is in-tree!